### PR TITLE
Add contact form for mobile version on Figma template 3

### PR DESCRIPTION
**Pull request by Erick-Osterling**
We did this contact form with pair programming. We used the formspree service to collect the user data and send it to an email address.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -177,9 +177,35 @@ <h2 class="hability">Skills</h2>
...
+    <p class="formp">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+
+    <form class="about_form" action="https://formspree.io/f/xrgrjovw" method="POST">
+    
+      <input type="text" name="fullname" class="formname" maxlength="30" required placeholder="Full name">
+      <input type="email" class="formemail" required placeholder="Email address">
+      <textarea maxlength="500" class="textholder" placeholder="Enter text here" required></textarea>
+      
+      <section class="finalsection">
+        <button class="submit_but">Get in touch</button>
```
Comment (ignatius22) [inline index.html]: - [x] Follow the guidelines by implementing the [type attribute](https://www.w3schools.com/tags/att_input_type_submit.asp), setting it to `submit`.

File: styles2.css [hunk 2]
```diff
@@ -443,3 +443,122 @@ Button interactions
...
+  padding: 12px;
+  width: 327px;
+  height: 114px;
+  font-size: 15px;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #60c095;
+}
+
+.submit_but {
```
Comment (ignatius22) [inline styles2.css:520]: - [x] Button interactions such as hover, active, etc, are missing, kindly provide one.

**Review by ignatius22 — CHANGES_REQUESTED** [review]
## Hi @Erick-Osterling, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] Please provide the correct demo link to this project, at least I don't see the contact form.
- [x] Please stick to the design templates in Figma.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Erick-Osterling) [reply index.html]: Thanks for the advice. Done.

File: styles2.css (hunk 2 above)
Comment (Erick-Osterling) [reply styles2.css:520]: Done. Sorry, I thought it was done. I have added a hover effect like in the rest of the buttons.

**Review by uwadonat — APPROVED** [review]
Hi **@ Erick-Osterling** , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
cong1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**